kubernetes/client.go

```diff
@@ -184,7 +184,7 @@ func (a *Adapter) NewInclusterConfigClientset(ctx context.Context) error {
 		IdleConnTimeout: idleConnTimeout,
 		Transport: &http.Transport{
 			Proxy:                 trCfg.Proxy,
-			DialContext:           trCfg.Dial,
+			DialContext:           cfg.Dial,
```

Comment:
no, that's basically dropping the work we try to achieve above: having timeouts configured for the client.

Reply:
ah no, hmm, maybe you are right, dunno :D

Reply:
Can you post some error that you try to fix?
Maybe we can also add a log to see if trCfg.Dial and cfg.Dial are different log with `"%p"` for example

Reply:
oh, then this needs a more detailed check considering the client-go update on `TransportConfig`.

Reply:
```
kubernetes/client.go:187:33: trCfg.Dial undefined (type *transport.Config has no field or method Dial)
```
this was the error, because client-go dropped `TransformConfig.Dial` [here](https://github.com/kubernetes/client-go/commit/5dab9a0b8438c16fe1572158c63b716e641ebeb4)

Reply:
Ok thanks, then go ahead like this